ksqldb-engine/src/main/java/io/confluent/ksql/util/QueryMetadataImpl.java

```diff
@@ -315,10 +314,6 @@ public void setCorruptionQueryError() {
     corruptionCommandTopic = true;
   }
 
-  protected boolean isClosed() {
```

Comment:
yeah -- I'm currently working on splitting this up into smaller PRs, I can move it out into a refactoring PR or just leave it alone if you think it might at some later time be useful (didn't seem to be right now)